src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1011,14 +1011,37 @@ DQLiftGatherSymCW::DQLiftGatherSymCW() {
         auto new_cvt_w = std::make_shared<ov::op::v0::Convert>(matched_out_w, ov::element::f16);
         auto gather_c = std::make_shared<ov::op::v0::Constant>(ov::element::i32, ov::Shape{}, 0);
         auto new_g_w = std::make_shared<ov::op::v8::Gather>(new_cvt_w, matched_out_ids, gather_c);
-        auto new_g_s = std::make_shared<ov::op::v8::Gather>(matched_out_s, matched_out_ids, gather_c);
+        auto new_scale_out = matched_out_s;
+
+        auto qcvtm_iter = node_to_output.find(qcvtm);
+        if (qcvtm_iter == node_to_output.end()) {
+            auto matched_qcvtm = qcvtm_iter->second.get_node_shared_ptr();
```

Comment:
How is it supposed to work?

if `qcvtm_iter == node_to_output.end()`, you can't access `qcvtm_iter->second` as this element doesn't exist?